kuma/wiki/kumascript.py

```diff
@@ -80,43 +83,21 @@ def _get_attachment_metadata_dict(attachment):
     }
 
 
-def get(document, cache_control, base_url, timeout=None):
-    """Perform a kumascript GET request for a document locale and slug."""
-    if not cache_control:
-        # Default to the configured max-age for cache control.
-        max_age = config.KUMASCRIPT_MAX_AGE
-        cache_control = 'max-age=%s' % max_age
+# TODO(djf): This get() function is actually implemented on top of
+# _post() and it performs an HTTP POST request. It should probably
+# be renamed to render_document(), and the post() method above should
+# be renamed to render_string(), maybe. For now, though, there are so
+# many tests that mock kumascript.get() that I've left the name unchanged.
+def get(document, base_url, timeout=config.KUMASCRIPT_TIMEOUT):
```

Comment:
I would vote for adding `cache_control` only to the headers, and not including it into the `env_vars` since @davidflanagan's mdn/kumascript#1035 (which this PR requires) doesn't need it in both places.